samples/unit-tests/maps/mapline-linewidth/demo.js

```diff
@@ -24,3 +24,45 @@ QUnit.test(
         );
     }
 );
+
+QUnit.test(
```

Comment:
Instead of adding a new test, please combine this one with the previous one (setting `lineWidth` to 30). And I think instead of loading the US map for the other one, we should just add that simple assertion into this new test, which is slimmer.